renderlib/CCamera.cpp

```diff
@@ -2,6 +2,81 @@
 
 #include "gesture/gesture.h"
 
+void
+CCamera::SetViewMode(const EViewMode ViewMode)
```

Comment:
this function was just moved from CCamera.h.  I think I also corrected a couple of the view vectors that hadn't been tested before.